Applications/WebPortal/WebApp/library/source/pages/account_page/entitlements_page/check_mark_button.tsx

```diff
@@ -9,27 +9,26 @@ interface Properties {
   /** Determines if the check mark is readonly or not. */
   isReadOnly: boolean;
 
-  /** The onClick event handler. */
+  /** The onClick event handler.
+   * This is not needed if the component is readonly.
```

Comment:
Don't bother describing the "This is not needed" part since that will be a property of all callbacks. All callbacks effectively have the property that if they're ignored or not used, nothing happens.